fixtures/large-error/src/lib.rs

```diff
@@ -0,0 +1,701 @@
+    Case91 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case92 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case93 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case94 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case95 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case96 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case97 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case98 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case99 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+
+    #[error("Important debug description of what went wrong, arg1: '{arg1}', arg2: '{arg2}', arg3: '{arg3}', arg4: '{arg4}'")]
+    Case100 {
+        arg1: ErrorPayload,
+        arg2: ErrorPayload,
+        arg3: ErrorPayload,
+        arg4: ErrorPayload,
+    },
+}
+
+impl LargeError {
+    pub fn discriminant(&self) -> u32 {
+        unsafe { *<*const _>::from(self).cast::<u32>() }
+    }
+}
+
+#[derive(Clone, Debug, Record)]
+pub struct ErrorWithContext {
+    pub error: LargeError,
+    pub context: String,
+}
+
+#[uniffi::export]
+pub fn error_message_from_error(error: &LargeError) -> String {
+    format!("{}", error)
```

Comment:
this and `error_code_from_error` fails to compile without the `BUF_SIZE` increment.